File: TransCoder_tokenized_test_set_functions/python/SORTING_ARRAY_REVERSE_AROUND_MIDDLE.py
def ifPossible ( arr, n ) :
    cp = [ 0 ] * n
    cp = arr
    cp . sort ( )
    for i in range ( 0, n ) :
        if ( not ( arr [ i ] == cp [ i ] ) and not ( arr [ n - 1 - i ] == cp [ i ] ) ) :
            return False
    return True

File: results/java_codex/SORTING_ARRAY_REVERSE_AROUND_MIDDLE.java
bool ifPossible(int arr[], int n) {
        int copy[n];
        for (int i = 0; i < n; i++) {
            copy[i] = arr[i];
        }
        sort(copy, copy + n);
        for (int i = 0; i < n; i++) {
            if (!(arr[i] == copy[i]) && !(arr[n - 1 - i] == copy[i])) return false;
        }
        return true;
    }